packages/interpretations/src/components/Avatar/UserAvatar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Avatar from '@material-ui/core/Avatar';
+import { withStyles } from '@material-ui/core/styles';
+import styles from './styles/Avatar.style';
+
+const UserAvatar = ({ classes, user }) => {
+    const initials = user.displayName
+        .split(" ")
```

Comment:
Have you tested how this will work/look if the user has multiple names in their display name?

Reply:
Here is what I get: 
![image](https://user-images.githubusercontent.com/6113918/51520865-22eb9b00-1e25-11e9-972b-3ebaff5746a2.png)

Reply:
Since we are getting the whole name through a string and do not know where the user's firstname and surname starts/end, i dont see any other option than: 
* take the first letter of the first word in first name and 
* first letter of the last name in surname:

![image](https://user-images.githubusercontent.com/13645152/51760217-9960f500-20d2-11e9-8722-a829afaadcb5.png)

Reply:
Can we also fetch first name and last name?